src/components/Modals/Receive/ReceiveInfo.tsx

```diff
@@ -316,10 +320,6 @@ export const ReceiveInfo = ({ asset, accountId, onBack }: ReceivePropsType) => {
             </HStack>
           </DialogFooter>
         </>
-      ) : (
-        <DialogBody alignItems='center' justifyContent='center'>
-          <Text translation='modals.receive.unsupportedAsset' />
-        </DialogBody>
```

Comment:
We were never hitting this and in case we ever did (which I did before removing `wallet` checks) that would be broken- v. old PR and this translation doesn't exist anymore: https://github.com/shapeshift/web/pull/451